naucse/views.py

```diff
@@ -13,6 +13,10 @@
 from naucse.urlconverters import register_url_converters
 from naucse.templates import setup_jinja_env
 
+import mkepub
+from bs4 import BeautifulSoup
```

Comment:
naucse už používá `lxml`, které by mělo stačit – další knihovna by neměla být potřeba. Ale jestli `lxml` neznáš, můžu to pak převést.